source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -0,0 +1,24 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+  {%- block body -%}
+
+    {%- set logo = logo|default({})|merge({html: html}) -%}
+
+    {%- with logo only -%}
+      {%- block logo -%}
+
+        {%- set image = image|default({})|merge({src: '../../images/logo.svg'}) -%}
```

Comment:
Looks like https://github.com/libero/pattern-library/pull/85#discussion_r278903614 was reverted. `include` works on non-Twig files too, so we can exclude the asset that way.